### ISSUE TITLE
NN is not accurate enough

### ISSUE THREAD

**Issue by kpegion** [kpegion/ensoNN#1]
My version of the NN is not 100% accurate in 100 epochs as they report for theirs. 

Mine is about 75% accurate. It does appear to converge at this accuracy.  

There's also some issues with the learning rate. They us an initial learning rate of 0.01 which is reduced by a factor of 0.5 after 50 epochs. I am not sure how to reduce the learning rate, but it does not seem to matter b/c mine converges at about 75% accurate with little change regardless of which rate I use.

Plan to test some sensitivity to some of the hyperparameters of the NN.

**Comment by kpegion** [comment]
Corrected after fixing output vetcor.
